changelogs/unreleased/4571-mobile-sidebar.yml

```diff
@@ -1,5 +1,5 @@
-change-type: patch
-description: "Enhancement bringing back functionality to close Sidebar when clicked outside of it on mobile"
+change-type: minor-improvement
```

Comment:
patch was correct, you have the template in the changelog folder if you need it. 
the "minor-improvement" is supposed to go in the "section" part :)

Reply:
Misunderstood :D